### ISSUE TITLE
Color strings are case sensitive in calls to setColor() MYR language builtin

### ISSUE THREAD

**Issue by theyoyojo** [isenseDev/MYR#108]
Reproduction:

```setColor("black")``` works

while on the other hand:

```setColor("Black")``` does not work.

However, a student unfamiliar to programming may be just trying to make like a duck or something and then get frustrated that the color "Black" doesn't exist. Who else thinks that color strings should be case insensitive?

**Comment by jasondkiesling** [comment]
We want to be careful about how we design the system. Real world systems/languages are case sensitive. We want MYR to be able to help users go into full languages beyond just the MYR API. Case sensitivity is something that many students have a hard time with moving on. We would be doing them a disservice by not using this as a teaching opportunity. When they see that their geometry is not actually black, they know to look at the line where they set the color to black to see if there is an issue. That is the idea behind the cursor. Where each line only does one thing, you can more easily see where your change didn't happen and look there to learn why it didn't.

**Comment by theyoyojo** [comment]
I just think the users might have more fun, which is also part of the point of this application. Many languages allow syntactic flexibility of some kind or another, and a beginner programmer has to learn to walk before they can run.

**Comment by jasondkiesling** [comment]
Although languages are flexible in syntax, the majority of programming languages are case sensitive. The idea of the cursor is how they learn to walk before they run. They only manipulate the cursor one part at a time. This allows them to easily read their code to see where their issue is. MYR is designed to be fun to use, but it is ultimately an educational tool. We want it to follow standard industry practices.

**Comment by theyoyojo** [comment]
Regardless of the value of case-insensitivity, I suggest a restoration of sanity to the current handling of this behavior via some fallback mechanism to handle invalid colors.

In production, MYR produces the following behavior:

![Selection_007](https://user-images.githubusercontent.com/1909841/54330916-fd416d80-45ed-11e9-848c-7a35217e6326.png)

It is not immediately obvious to the user than an error has occurred. The IDE currently only complains about the lack of a semicolon.

The exact same non-error behavior is observed when the valid string "white" is passed to setColor()

![Selection_010](https://user-images.githubusercontent.com/1909841/54331091-be5fe780-45ee-11e9-9284-48859ab06461.png)

Or any random string for that matter:

![Selection_011](https://user-images.githubusercontent.com/1909841/54331105-ca4ba980-45ee-11e9-800e-65ff8c8c2e19.png)

tl;dr: Invalid paramaters produce the same behavior as a valid parameter in the setColor() function

**Comment by theyoyojo** [comment]
Regarding my original concern, sniff this diff:

```
diff --git a/src/myr/Myr.js b/src/myr/Myr.js
index 01669a9..dd500e0 100644
--- a/src/myr/Myr.js
+++ b/src/myr/Myr.js
@@ -215,7 +215,7 @@ class Myr {
   };
 
   setColor = (color) => {
-    this.color = color;
+    this.color = color.toLowerCase() ;
   }
 
   getRandomColor = () => {

**Comment by theyoyojo** [comment]
Regarding the duck, I have made substantial progress and look forward to General Availability in about 30ish minutes: https://learnmyr.org/o8UTFYHcTnSxjicpaLjO

**Comment by jasondkiesling** [comment]
By design it is case-sensitive and should stay as such. Our goal is to be fail resistant, not to be 100% fail safe. The current fallback mechanism is that the color goes to white.
